proto/scripts/fix_remote_py.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
 set -euo pipefail
 
+root="../runtimes/pythonrt/runner/pb"
+
 # Make directories in pb importable
-find ../runtimes/pythonrt/runner/pb -type d -exec touch {}/__init__.py \;
+find ${root} -type d -exec touch {}/__init__.py \;
 
 # Change import path in grpc
-sed 's/from autokitteh.remote.v1 import remote_pb2/from . import remote_pb2/' ../runtimes/pythonrt/runner/pb/autokitteh/remote/v1/remote_pb2_grpc.py > /tmp/fixed.py
-mv /tmp/fixed.py ../runtimes/pythonrt/runner/pb/autokitteh/remote/v1/remote_pb2_grpc.py
+
+replace() {
```

Comment:
See like you need `-i` and it'll work both on OSX and Linux (see [here](https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux))

Reply:
i don't follow - i did this because `sed -i` does not work on both linux and os x. event the link you included mentions this?

Reply:
They have a solution in the link to something that works on both.